src/lib.rs

```diff
@@ -575,7 +584,8 @@ impl Regex {
         } else if r == onig_sys::ONIG_MISMATCH {
             None
         } else {
-            panic!("Onig: Internal error during regex search");
+            let Error { description, .. } = Error::from_code(r);
```

Comment:
Top pattern matching! However maybe consider:
```rust
let err = Error::from_code(r);
panic!("...", err.description());
```

Reply:
Yeah, you're right. But I think in #80 that code changed anyway, so if you merge that then it's all good.